linux/check_quiesce_snapshot_custom_script/check_quiesce_snapshot_custom_script.yml

```diff
@@ -21,7 +21,10 @@
           vars:
             skip_msg: "Skip testcase '{{ ansible_play_name }}', it is not supported by {{ guest_os_ansible_distribution }}"
             skip_reason: "Not Supported"
-          when: guest_os_ansible_distribution == 'FreeBSD'
+          when: >-
+            (guest_os_ansible_distribution == 'FreeBSD' or
```

Comment:
Is it  (guest_os_ansible_distribution == 'FreeBSD') or
             (guest_os_ansible_distribution == 'Flatcar' and
              guest_os_ansible_distribution_ver is version('3760.2.0', '>='))?

Reply:
No. It's my coding habit to put conditions with 'or' into parentheses.